File: src/test/java/org/hisp/dhis/rules/models/RuleActionHideFieldTest.java
package org.hisp.dhis.rules.models;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.junit.runners.JUnit4;

import static org.junit.Assert.assertEquals;

@RunWith( JUnit4.class )
public class RuleActionHideFieldTest
{
    @Test
    public void createMustSubstituteEmptyStringIfArgumentsNull()
    {
        RuleActionHideField ruleActionHideField =
            RuleActionHideField.create( null, "test_field" );

        assertEquals( "" , ruleActionHideField.content() );
    }
}

File: src/test/java/org/hisp/dhis/rules/models/RuleActionHideFieldTest.kt
package org.hisp.dhis.rules.models

import kotlin.test.assertEquals
import kotlin.test.Test

class RuleActionHideFieldTest {
    @Test
    fun createMustSubstituteEmptyStringIfArgumentsNull() {
        val ruleActionHideField = RuleActionHideField("test_field")
        assertEquals("", ruleActionHideField.content)
    }
}
